src/api/config/feature.yml

```diff
@@ -0,0 +1,2 @@
+features:
+  image_templates: true
```

Comment:
Now we have a new backend/repository for the "feature" gem that have the default values in there, so if the file doesn't have a key (because misconfiguration) then it will still have the default value defined there in the file `lib/feature_switch/obs_repository.rb` as the `DEFAULTS` constant. 

Speak up if you wan to have it in another way.